mlkem/sys.h

```diff
@@ -65,17 +65,21 @@
 #if defined(__STDC_VERSION__) && __STDC_VERSION__ >= 199901L
 #define INLINE inline
 #define ALWAYS_INLINE __attribute__((always_inline))
+#define NO_INLINE __attribute__((noinline))
 #elif defined(_MSC_VER)
 #define INLINE __inline
 #define ALWAYS_INLINE __forceinline
+#define NO_INLINE
```

Comment:
It looks like `noinline` is also supported by MSVC: `__declspec(noinline)`

Reply:
Do we have any CI in Windows to verify that this won't break anything?